Backend/Hub/AuthHub.cs

```diff
@@ -100,6 +126,10 @@ public async Task<bool> ChangeUsername(string username)
             await Database.SaveChangesAsync();
             await Clients.Group(user.GroupMember.Group.SignalRId).UpdateGroupMember(new(user.Id.ToString(), username, user.GroupMember.IsHost, user.GroupMember.IsDone, user.Score));
             await Clients.Caller.UpdateUserData(username, user.Score);
+
+            //Add to sessionlog
+            var entry = new SessionLogEntry(SessionLogEntryType.ChangeUsername, user.Id + "changed username to " + username, DateTime.UtcNow, user);
```

Comment:
Maybe here you should store the user's old username since we already keep track of the newest one.